docs/getting-started.md

```diff
@@ -7,283 +7,209 @@ path: /docs/getting-started/
 
-You may have noticed that the button background, as well as the label and underline on focused text
-input fields, defaults to the Indigo 500 (`#673AB7`) color from the [Material Design color palette](https://material.io/guidelines/style/color.html#color-color-palette).
-This is part of the default theme that ships with MDC Web; it uses Indigo 500 for a primary color, and
-Pink A200 (`#FF4081`) for a secondary color. Let's change the theme's primary color.
+.foo-button {
+  @include mdc-button-ink-color(teal);
+  @include mdc-states(teal);
+}
+```
+[@material/button has documentation about the required HTML of a button](https://github.com/material-components/material-components-web/tree/master/packages/mdc-button). Update your index.html to include this HTML, and add the foo-button class onto the element:
 
-A common misconception when implementing Material Design is that the colors you use _must_ come from
-the Material Design color palette. This is not true at all. The only defining guideline for color within Material
-Design is that it has "bold hues juxtaposed with muted environments, deep shadows, and bright
-highlights". Let's change our theme's primary color to `#0E4EAD`, the "Afternoon_Skyblue" color from
-the [Deep_Skyblues Colourlovers Palette](http://www.colourlovers.com/palette/334208/Deep_Skyblues).
+```
+<body>
+  <button class="foo-button mdc-button">
+    Button
+  </button>
+</body>
+```
 
-The easiest way to change the theme of an MDC Web application is via [CSS Variables](https://developer.mozilla.org/en-US/docs/Web/CSS/Using_CSS_variables). Simply add the
-following to the `<head>` tag of `index.html`:
+You also need to configure the sass-loader to understand the @material syntax. Update your webpack.config.js by changing `{ loader: 'sass-loader' }` to :
 
-```html
-<style>
-  :root {
-    --mdc-theme-primary: #0e4ead;
+```
+{
+  loader: 'sass-loader',
+  options: {
+    importer: function(url, prev) {
+      if(url.indexOf('@material') === 0) {
+        var filePath = url.split('@material')[1];
+        var nodeModulePath = `./node_modules/@material/${filePath}`;
+        return { file: require('path').resolve(nodeModulePath) };
+      }
+      return { file: url };
+    }
   }
-</style>
+}
 ```
 
-If you're using any supported browser, besides IE 11, you'll see that the button background as well as the focused underline and label on text
-fields are now a nice, dark shade of blue.
+Now run npm start again and open localhost:8080. You should see the a Material Design button!
+
+![Button](button.png?raw=true)
 
-> Note that using CSS Variables is just one way of theming using MDC Web. Check out our
-[theming documentation](./theming.md) for more info.
+### Step 3: Webpack with ES2015
 
-### Finishing touches: adding custom styles
+We need to configure webpack to bundle ES2015 JavaScript into standard JavaScript, through [babel](https://babeljs.io/learn-es2015/). You’ll need all of these dependencies:
```

Comment:
Is there a reason this is linking to Babel's "Learn ES2015" page specifically?

Reply:
hmm nope. changed it back